### DISCUSSION TITLE
jsonTable styles interne Tabellen-Elemente beeinflussen

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2243] [Q&A]
![image](https://user-images.githubusercontent.com/84818934/220757490-113d9e95-0940-4a68-8461-c3941bb00c76.png)

Zurzeit kann man mit Styles-Eintrag die Elemente in der internen Tabelle von einer jsonTable beeinflussen.
```
.jarvis-widget-9dc6b202-f067-4dd6-8090-b12f9fd427ba td {
	white-space: break-spaces;
}
```
Kann man diese Elemente auch mit einem styles Eintrag direkt in der jsonTable so einstellen, wie mit dem Eintrag in Styles?

Der "classes"-Parameter greift nur auf der darüberliegenden Ebene. Wie muss man es einstellen, damit auch die in dem Element befindliche Tabelle beeinflusst wird?

Hab es mal mit 
```
/* In den Styles definiert */
.whitespaceBS td {
	white-spaces: break-spaces !important;
}
```
`"classes": "whitespaceBS:has(div.td)",` probiert 
![image](https://user-images.githubusercontent.com/84818934/220760620-9be2a1b2-0256-438b-b052-c6061af151ad.png)

**Comment by Zefau** [comment]
`whitespaceBS:has(div.td)` ist keine Klasse, sondern `whitespaceBS`. Geht das?

**Reply by mcuiobroker** [reply]
Nein. Eine Idee wie man das anpacken muss / kann? Vielleicht geht es auch gar nicht?
